packages/generator-open-wc/generators/building-webpack/templates/static/index.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<html>
+  <head></head>
+  <body>
+    <my-app></my-app>
```

Comment:
Is there a way to make this less hardcoded? @daKmoR

Reply:
sure that would be a scaffold generator (take a look at the testing one)... so this also means that this should probably be split in two generator :p